src/components/Components/ComponentDetails/__tests__/ComponentBuildSettings.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import usePACState, { PACState } from '../../../../hooks/usePACState';
+import { getHelpPopoverText, getBuildTriggerText } from '../tabs/ComponentBuildSettings';
+
+jest.mock('../../../../hooks/usePACState');
+jest.mock('../../../../../src/components/HelpPopover');
+
+const usePacStateMock = usePACState as jest.Mock;
```

Comment:
mocks shouldn't be required here since only utils are being tested.

Reply:
Mock removed